client/grpc-web/package.json

```diff
@@ -27,7 +27,6 @@
     "dist"
   ],
   "peerDependencies": {
-    "@types/google-protobuf": "^3.2.5",
```

Comment:
This package provides TypeScript definitions which are only required during compilation (and already satisfied as a devDependency); consumers do not need to fulfil this dependency as they are not expected to compile this package.